lib/child-writer/writer-proxy.js

```diff
@@ -126,7 +127,7 @@ return isElevated().then((elevated) => {
       name: packageJSON.displayName
     }).then((stdout, stderr) => {
       if (!_.isEmpty(stderr)) {
-        throw new Error(stderr);
+        throw errors.createError(stderr);
```

Comment:
Not really sure. The purpose of this block is to bridge errors coming from the child script, which should already contain enough context.